### DISCUSSION TITLE
Is Ember still not recommended or does it support multiprotocol now?

### DISCUSSION THREAD

**Discussion by NewsGuyTor** [Koenkk/zigbee2mqtt#22811] [General]
I have a SkyConnect with multiprotocol that works fine, but there are a lot of messages everywhere to switch to Ember. However, I've heard that it's still in alpha phase - that it hasn't implemented proper multiprotocol support yet. Is that accurate?

I'm obviously not going to pollute the environment purposefully because of a software update.

**Comment by francisp2** [comment]
multiprotocol is no longer recommended by nabu casa, and no, the ember firmware does not support multiprotocol

**Comment by NewsGuyTor** [comment]
EZSP will continue to work fine right? Updating Z2M won't brick our setups?

**Reply by francisp2** [reply]
No, EZSP stays, it just don't get any updates more.

**Comment by Koenkk** [comment]
Ember is not in alpha phase anymore, its fully supported. It replaces the ezsp driver which will not be updated anymore (but will stay available). Multiprotocol firmware is not recommended for both ember and ezsp
